TCMSApp/src/client/app/tests/tests-list.html

```diff
@@ -0,0 +1,172 @@
+<div class="matter">
+  <div class="container">
+
+<!-- header -->
+    <div class="clearfix">
+      <h1>
+        Tests
+        <form class="form-inline pull-right">
+          <input type="text" class="form-control" placeholder="Filter">
+        </form>
+      </h1>
+    </div>
+    <hr>
+
+<!--main content-->
+    <div class="row">
+
+<!-- suites -->
+      <div class="col-xs-3">
+        <div class="panel panel-info">
+          <div class="panel-heading">
+            <h3>Suites <button type="button" class="btn btn-primary btn-xs" ng-click="vmTests.openAddSuite()">NEW</button></h3>
+          </div>
+          <div class="table-responsive">
+            <table class="table table-hover" style="font-size:larger">
+              <tbody>
+                <tr ng-repeat="s in vmTests.f track by $index" 
+                    ng-click="seeSuite = !seeSuite; vmTests.setSuite(s)" 
+                    style="cursor: pointer" ng-class="{active: vmTests.compareObj(vmTests.currentSuite, s) }">
```

Comment:
rather expensive part. to display active suite we may stringify all objects. and stringify `vmTests.currentSuite` n times.
can we just compare ID's of suites? :)